_data-prepper/simple-pipelines.md

```diff
@@ -0,0 +1,149 @@
+```
+docker run --name data-prepper -p 4900:4900 -v ${PWD}/pipelines.yaml:/usr/share/data-prepper/pipelines.yaml -v ${PWD}/data-prepper-config.yaml:/usr/share/data-prepper/data-prepper-config.yaml opensearchproject/data-prepper:latest
+```
+
+You should see log output and after a few seconds some UUIDs in the output. It should look something like the following:
+
+```
+2021-09-30T20:19:44,147 [main] INFO  org.opensearch.dataprepper.pipeline.server.DataPrepperServer - Data Prepper server running at :4900
+2021-09-30T20:19:44,681 [random-source-pool-0] INFO  org.opensearch.dataprepper.plugins.source.RandomStringSource - Writing to buffer
+2021-09-30T20:19:45,183 [random-source-pool-0] INFO  org.opensearch.dataprepper.plugins.source.RandomStringSource - Writing to buffer
+2021-09-30T20:19:45,687 [random-source-pool-0] INFO  org.opensearch.dataprepper.plugins.source.RandomStringSource - Writing to buffer
+2021-09-30T20:19:46,191 [random-source-pool-0] INFO  org.opensearch.dataprepper.plugins.source.RandomStringSource - Writing to buffer
+2021-09-30T20:19:46,694 [random-source-pool-0] INFO  org.opensearch.dataprepper.plugins.source.RandomStringSource - Writing to buffer
+2021-09-30T20:19:47,200 [random-source-pool-0] INFO  org.opensearch.dataprepper.plugins.source.RandomStringSource - Writing to buffer
+2021-09-30T20:19:47,704 [random-source-pool-0] INFO  org.opensearch.dataprepper.plugins.source.RandomStringSource - Writing to buffer
+2021-09-30T20:19:48,207 [random-source-pool-0] INFO  org.opensearch.dataprepper.plugins.source.RandomStringSource - Writing to buffer
+2021-09-30T20:19:48,677 [random-source-pool-0] INFO  org.opensearch.dataprepper.plugins.source.RandomStringSource - Writing to buffer
+2021-09-30T20:19:49,179 [random-source-pool-0] INFO  org.opensearch.dataprepper.plugins.source.RandomStringSource - Writing to buffer
+2021-09-30T20:19:49,181 [simple-test-pipeline-processor-worker-1-thread-1] INFO  org.opensearch.dataprepper.pipeline.ProcessWorker -  simple-test-pipeline Worker: Processing 6 records from buffer
+07dc0d37-da2c-447e-a8df-64792095fb72
+5ac9b10a-1d21-4306-851a-6fb12f797010
+99040c79-e97b-4f1d-a70b-409286f2a671
+5319a842-c028-4c17-a613-3ef101bd2bdd
+e51e700e-5cab-4f6d-879a-1c3235a77d18
+b4ed2d7e-cf9c-4e9d-967c-b18e8af35c90
+```
+
+In another terminal, you can access the Data Prepper server API.
+
+```
+curl https://localhost:4900/list
+```
+
+It will output a JSON response with the current pipelines:
+
+```
+{"pipelines":[{"name":"simple-test-pipeline"}]}
+```
+
+
+## Shut down Data Prepper
+
+You can shutdown Data Prepper either by using the shutdown API, or stopping the Docker process.
+
+```
+curl -X POST https://localhost:4900/shutdown
+```
+
+## Adding a processor
+
+The sample above just demonstrates the bare minimum a pipeline can have: A source sending data to a sink. The following example adds a string converter processor. This simple processor transforms the string by making it uppercase.
```

Comment:
```suggestion
The previous example demonstrates the bare minimum a pipeline can have: a source sending data to a sink. The following example adds a string converter processor. This simple processor transforms the string by making it uppercase:
```

Reply:
Use "above" only for physical space or screen descriptions. For orientation within a document use "previous," "preceding," or "earlier."

Reply:
Are we referring to the most recent example? If not, we need to clarify which example we are referencing.

Reply:
@oeyh Can you please clarify which example this is referring to?